Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -101,13 +104,29 @@ public struct BPKPrice: View {
     
     @ViewBuilder
     private var priceLabel: some View {
-        BPKText(price, style: priceFontStyle)
+        switch alignment {
+        case .leading, .row:
+            BPKText(price, style: priceFontStyle)
+        case .trailing:
+            HStack(spacing: .sm) {
+                BPKText(price, style: priceFontStyle)
+                redirectingIcon
+            }
+            Spacer().frame(height: 0)
```

Comment:
Add this to remove an extra space between HStack and Text below caused by adding icon